redis_keys.go

```diff
@@ -4,12 +4,12 @@ const (
 	connectionsKey                   = "rmq::connections"                                           // Set of connection names
 	connectionHeartbeatTemplate      = "rmq::connection::{connection}::heartbeat"                   // expires after {connection} died
 	connectionQueuesTemplate         = "rmq::connection::{connection}::queues"                      // Set of queues consumers of {connection} are consuming
-	connectionQueueConsumersTemplate = "rmq::connection::{connection}::queue::[{queue}]::consumers" // Set of all consumers from {connection} consuming from {queue}
-	connectionQueueUnackedTemplate   = "rmq::connection::{connection}::queue::[{queue}]::unacked"   // List of deliveries consumers of {connection} are currently consuming
+	connectionQueueConsumersTemplate = "rmq::connection::{connection}::queue::{{queue}}::consumers" // Set of all consumers from {connection} consuming from {queue}
+	connectionQueueUnackedTemplate   = "rmq::connection::{connection}::queue::{{queue}}::unacked"   // List of deliveries consumers of {connection} are currently consuming
 
 	queuesKey             = "rmq::queues"                     // Set of all open queues
-	queueReadyTemplate    = "rmq::queue::[{queue}]::ready"    // List of deliveries in that {queue} (right is first and oldest, left is last and youngest)
-	queueRejectedTemplate = "rmq::queue::[{queue}]::rejected" // List of rejected deliveries from that {queue}
+	queueReadyTemplate    = "rmq::queue::{{queue}}::ready"    // List of deliveries in that {queue} (right is first and oldest, left is last and youngest)
+	queueRejectedTemplate = "rmq::queue::{{queue}}::rejected" // List of rejected deliveries from that {queue}
```

Comment:
This is a breaking change which is hard to deal with when using a setup which relies on `[]` (like we do internally). Do you think you could add a way to have `{}` by default, but switch back to using `[]` with some sort of flag? 🤔

Reply:
Yeah sure, I will see what I can do about this one

Reply:
@wellle  Might know where I can find the setup which relies on [] ? Thanks

Reply:
I'll look into this 👍

Reply:
Actually I found it already. We use this feature internally: https://github.com/twitter/twemproxy/commit/f08088f7ed8a370a45365723ba64fb6fc463a0b9

In there they use `hash_tag: "{}"` to demonstrate. In our specific setup we use it like this instead: hash_tag: "[]".

So ideally we would be able to support both of these use cases (with `{}` being the default).

Reply:
Here's an idea of how it could be done. I don't like how it uses a global variable, so please let me know if you see a better approach. 🙏 

1. Add a global `rmq.HashTag = "{}"`. This allows users (like us) to set `rmq.HashTag = "[]"` before opening an rmq connection.

2. Then in `newQueue()` we currently have code like this:
    ```go
    readyKey := strings.Replace(queueReadyTemplate, phQueue, name, 1)
    ```
    Let's replace that like this:
    ```go
    readyKey := queueReadyKey(name)
    ```
    So in `redis_keys.go` we would add these functions which take the `rmq.HashTag` value into account. For example it may look like this (untested):
    ```go
    func queueReadyKey(queueName string) string {
    	hashTagOpen, hashTagClose := rmq.HashTag[0], rmq.HashTag[1]
    	return fmt.Sprintf("rmq::queue::%s%s%s::ready", hashTagOpen, queueName, hashTagClose)
    }
   ```

Do you think you could try this for the 4 keys which use the `{queue}` placeholder?